src/utils.rs

```diff
@@ -719,7 +719,7 @@ fn str_width(s: &str) -> usize {
 }
 
 #[cfg(feature = "ansi-parsing")]
-fn char_width(c: char) -> usize {
+pub fn char_width(c: char) -> usize {
```

Comment:
If we're going to make this `pub`, please fold the `not(feature = "ansi-parsing")` variant into this function (which I think happens implicitly anyway?) and add a docstring.

Reply:
Sorry, I didn't consider it carefully. I have now made this function pub(crate).

Reply:
So you don't need it in dialoguer anymore?